tasks/app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,19 @@
+<html>
+<head>
+  <link href='https://fonts.googleapis.com/css?family=Rock+Salt' rel='stylesheet' type='text/css'>
+</head>
+<body>
+    <h1>Task List</h1>
+    <div class="center">
+      <% @tasks.each do |task| %>
+        <p>
+          <% if !task.completed.nil?  %>
+            <a href="/tasks/<%= task.id %>"><%= task.name %></a>
```

Comment:
This all of the data is the same except for the class information, I might recommend putting only the piece about the class inside of the conditional and the rest outside.